.travis.yml

```diff
@@ -4,11 +4,11 @@ language: cpp
 
 env:
   matrix:
-    - OS_TYPE=debian10 CMAKE_BUILD_TYPE=Debug   SONAR_SCANNER=OFF COVERALLS=OFF NON_STOCK_CPPZMQ=ON TANGO_USE_USING_NAMESPACE=OFF USE_PCH=ON
-    - OS_TYPE=debian10 CMAKE_BUILD_TYPE=Release SONAR_SCANNER=OFF COVERALLS=OFF NON_STOCK_CPPZMQ=ON
-    - OS_TYPE=debian9  CMAKE_BUILD_TYPE=Debug   SONAR_SCANNER=OFF COVERALLS=OFF NON_STOCK_CPPZMQ=ON
-    - OS_TYPE=debian8  CMAKE_BUILD_TYPE=Debug   SONAR_SCANNER=ON  COVERALLS=ON  NON_STOCK_CPPZMQ=OFF
-    - OS_TYPE=debian7  CMAKE_BUILD_TYPE=Debug   SONAR_SCANNER=OFF COVERALLS=OFF NON_STOCK_CPPZMQ=ON
+    - OS_TYPE=debian10 CMAKE_BUILD_TYPE=Debug   SONAR_SCANNER=OFF COVERALLS=OFF STOCK_CPPZMQ=ON TANGO_USE_USING_NAMESPACE=OFF USE_PCH=ON
+    - OS_TYPE=debian10 CMAKE_BUILD_TYPE=Release SONAR_SCANNER=OFF COVERALLS=OFF STOCK_CPPZMQ=ON
+    - OS_TYPE=debian9  CMAKE_BUILD_TYPE=Debug   SONAR_SCANNER=OFF COVERALLS=OFF STOCK_CPPZMQ=ON
+    - OS_TYPE=debian8  CMAKE_BUILD_TYPE=Debug   SONAR_SCANNER=ON  COVERALLS=ON  STOCK_CPPZMQ=OFF
```

Comment:
I would use USE_PCH=ON on Debian 8 as well since we got a timeout on one of the Travis builds for this Debian 8 config.

Reply:
I can make a followup PR and always have USE_PCH=ON as #567 requests?

Reply:
> I can make a followup PR and always have USE_PCH=ON as #567 requests?

Sure!
And as you suggested in #567, we should keep one with PCH=OFF to ensure that it still works in this case.